tests/acceptance/pageObjects/FilesPageElement/publicLinksDialog.js

```diff
@@ -43,9 +196,28 @@ module.exports = {
           })
         })
       return Promise.all(promiseList)
+    },
+    assertErrorMessageForPublicLinks: function (errorMessage) {
+      return this
+        .waitForElementVisible('@filesSidebarError')
+        .getText(
+          this.elements.filesSidebarError.locateStrategy,
+          this.elements.filesSidebarError.selector,
+          (result) => {
+            assert.strictEqual(result.value, errorMessage)
+          }
+        )
+    },
+    assertPublicLinkNotCreated: function () {
+      return this
+        .waitForElementVisible('@createLinkButton')
```

Comment:
How does this function asserts that public link is not created?

Reply:
:om: